### DISCUSSION TITLE
Testing the Z-Station Z-wave>me multi-protocol USB adapter - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#52] [Comments]
# Testing the Z-Station Z-wave>me multi-protocol USB adapter - Haade.fr

I am going to test this dongle referenced ZMEEZSTATION_ZW_ZB from the Z-Wave brand>me the Z-station USB adapter and see if it meets the opensource requirements Zigbee2mqtt and ZwaveJS UI among others

https://haade.fr/en/blog/test-z-station-z-wave-me-multi-protocol-zigbee-z-wave-thread-matter-bluetooth-adaptor-swiss

**Comment by zalwandri78** [comment]
What types of serial interfaces does the Z-Station provide for communication? regard <a href=" https://jakarta.telkomuniversity.ac.id/menyelami-dunia-botnet-ancaman-tersembunyi-di-era-digital/ "> Tel U </a>

**Comment by PackElend** [comment]
Hey Nico,
how do yout think about the device today.
I have learned there were some firmware updates last year. 

thx
stefan

**Reply by haade-administrator** [reply]
Hi, it's true that I haven't used this module for a long time, I'm going to do the updates and I'll give you an update, there are improvements on the zigbee and the z-wave long range

**Reply by PackElend** [reply]
Great for doing that, looking forward to seeing the results.

By the way could you simplify the zigbee modes.

I have currently z-wave and Bluetooth and going to get some zigbee for lighting, type not defined yet

**Reply by PackElend** [reply]
Have you got any chance to test it again?
